src/components/icons-with-counter/IconsWithCounter.style.tsx

```diff
@@ -0,0 +1,14 @@
+export const styles = {
+  iconBox: {
+    width: 'max-content',
+    display: 'flex',
+    alignItems: 'centre'
```

Comment:
`alignItems: 'center'`

Reply:
thanks

Reply:
> thanks

you are welcome, but, looks like you forgot to fix that 😅

Reply:
yeah 😅